test/db/cmd/tables

```diff
@@ -0,0 +1,136 @@
+NAME=tables sum
+FILE=bins/elf/analysis/elf-virtualtable
+CMDS=<<EOF
+f, size/sum
+EOF
+EXPECT=<<EOF
+size
+----
+16.8K
+
+EOF
+RUN
+
+NAME=tables sort
+FILE=bins/elf/analysis/elf-virtualtable
+CMDS=<<EOF
+f, size/cols,size/sort,size/uniq
+EOF
+EXPECT=<<EOF
+size
+----
+0
+1
+2
+3
+4
+6
+8
+9
+13
+16
+17
+21
+23
+24
+26
+28
+32
+36
+42
+43
+52
+56
+62
+64
+66
+68
+80
+88
+96
+101
+104
+124
+158
+176
+203
+240
+256
+268
+272
+432
+448
+464
+497
+512
+540
+978
+1160
+1255
+2424
+4132
+
+EOF
+RUN
+
+NAME=tables csv
+FILE=bins/elf/analysis/elf-virtualtable
+CMDS=<<EOF
+f, size/ne/0,addr/ne/0x00000000,/head/10,:csv~:..20
```

Comment:
Is space here intentional while the rest isn't? @ret2libc will it be problematic with the newshell?

Reply:
the space is intentional. and tests are passing in newshell. but still will be better to have all those test with a space if any of them doesnt have it for compat